.github/workflows/pull_request.yml

```diff
@@ -66,6 +67,7 @@ jobs:
       - uses: actions/setup-python@v4
         with:
           python-version: ${{ env.PYTHON_VERSION }}
+          cache: 'pipenv'
       - name: Install apt dependencies
```

Comment:
Why do we install the apt dependencies multiple times?. Only needed for the unit tests.

Reply:
Removed, only installing for unit test now.